wiki/604838-SubmissionDashboards.md

```diff
@@ -13,9 +13,31 @@ This section provides dashboards to the person who submit NLP Sandbox tools for
 - You can cancel a submission tthat has not started yet.
 - Only submissions that have successfully completed are counted toward your (or your team's) submission quota.
 - If you can't figure out why your submission failed, please [open a ticket].
+- The `SubmissionLogs` column contains information of the workflow that is ran per submission. It doens't actually contain logs generated by your submissions. If your submission is INVALID or REJECTED, you will have to request for logs.
 
-{column}
-{row}
+### Submission Status Definitions
+
+The column `Status` of the dashboard takes the following values: 
+
+Status | Description
+---|---
+RECEIVED | Your submission has been received
+EVALUATION_IN_PROGRESS | Determining internal queue to submit to and evaluation submission on i2b2 dataset
+ACCEPTED | Submitted to different sites for submissions
+INVALID | Model either failed to run on i2b2 data, or prediction file is invalid
+REJECTED | Model either failed to run on all internal site's data, or prediction file(s) is invalid
+
+## Evaluation on other sites
+
+`{site} Status` column
```

Comment:
Technically i2b2 is not a data site but at the same time we have not mentioned in our documentation "Sage data site". From a user perspective, it may be easier to refer to "{Dataset} Status" here rather than Site. When needed, we can remind users that a dataset is hosted at a specific data site, e.g. to justify that logs are not returned. The proposed change is also more friendly towards data site that may provided more than one concurrent dataset (e.g. Sage data site hosting i2b2 and Mayo Clinic dataset).

Reply:
I split it up this way, because there are different status definitions. All docker containers are run on the i2b2 dataset prior to being run on any other datasets.  Not sure how we should document that.

Reply:
OK, I see now that there is a section for "status" and another section describing "{site} status". Still wondering how we would do when one site will provide more than one datasets. As mentioned Mayo Clinic is thinking to let Sage host the dataset (no sensitive information involved). We can cross this bridge when we are there.